### PR TITLE
Add desired state to workbench instances.

### PR DESCRIPTION
**Pull request by bcreddy-gcp**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
workbench: added `desired_state` field to `google_workbench_instance` resource
```

### CODE REVIEW

**Comment by bcreddy-gcp** [comment]
@ScottSuarez, Could you update the IAM roles on the service account specified in full test that were required for the test to pass?
Could you test/review the PR when you get a chance @slevenick. It'd be great to have this change in the immediate release if possible.

File: mmv1/templates/terraform/post_create/workbench_instance.go.erb [hunk 1]
```diff
@@ -1,3 +1,13 @@
 if err := waitForWorkbenchInstanceActive(d, config, d.Timeout(schema.TimeoutCreate) - time.Minute); err != nil {
   return fmt.Errorf("Workbench instance %q did not reach ACTIVE state: %q", d.Get("name").(string), err)
 }
+
+if p, ok := d.GetOk("desired_state"); ok && p.(string) == "STOPPED" {
```
Comment (bcreddy-gcp) [inline mmv1/templates/terraform/post_create/workbench_instance.go.erb:5]: here's the state modification post_create code

Comment (slevenick) [reply mmv1/templates/terraform/post_create/workbench_instance.go.erb:5]: Ah ok, I must have missed this!

**Review by slevenick — APPROVED** [review]

**Comment by slevenick** [comment]
Can you add a release note for this in the original PR description?

**Comment by bcreddy-gcp** [comment]
Added the release note.

**Pull request merged**